addons/mail/models/mail_mail.py

```diff
@@ -576,6 +576,14 @@ def send(self, auto_commit=False, raise_exception=False):
             finally:
                 if smtp_session:
                     smtp_session.quit()
+        return {
```

Comment:
This should not apply when clicking the button from list view.

Reply:
Oh yes correct, will look into that issue.